docs/operating-scylla/nodetool-commands/removenode.rst

```diff
@@ -27,19 +29,16 @@ Example:
 
     nodetool removenode 675ed9f4-6564-6dbd-can8-43fddce952gy
 
-Note that all the nodes in the cluster participate in the ``removenode`` operation to sync data if needed. For this reason, the operation will fail if one or more nodes in the cluster are not available.
-In such a case, to ensure that the operation succeeds, you must explicitly specify a list of unavailable nodes with the ``--ignore-dead-nodes`` option.
 
-Use a comma-separated list to specify the Host IDs of all unavailable nodes in the cluster before specifying the node to remove.
+.. _removenode-ignore-dead-nodes:
 
-.. warning::
-    Instead of Host IDs, you can specify the IP addresses of all unavailable nodes. However, this feature is deprecated
-    with consistent topology updates enabled and will be removed in a future release. Using Host IDs is recommended.
+Ignoring Dead Nodes
+---------------------
 
-.. warning::
-    Before you use the ``nodetool removenode --ignore-dead-nodes`` command, you must make sure that both the node you want
-    to remove AND the nodes you want to ignore are permanently down and cannot be recovered. Ignoring a node that is running
-    or a node that is temporarily down and could be later restored in the cluster may result in data loss.
+All the nodes in the cluster participate in the ``removenode`` operation to sync data if needed. For this reason, the operation will fail if one or more nodes in the cluster are not available.
+In such a case, to ensure that the operation succeeds, you must explicitly specify a list of unavailable nodes with the ``--ignore-dead-nodes`` option.
```

Comment:
We should also inform that the ignored nodes are permanently banned --- the same as removed node.

Reply:
Also similar caveat needs to be added to replace procedure --- the replaced node is permanently banned at the beginning! (Think of replace as removenode combined with bootstrap, happening at the same time)